src/dps-email-poller/src/test/java/ca/bc/gov/open/pssg/rsbc/dps/dpsemailpoller/scheduler/EmailPollerJunkRemovalTest.java

```diff
@@ -91,23 +96,23 @@ public void with5JunkEmailShouldBeRemoved() throws Exception {
 
         List<EmailMessage> result = new ArrayList<>();
 
-        EmailMessage item = new EmailMessage(exchangeServiceMock);
-        item.setSubject(I_M_JUNK);
+        for(int i = 0; i < 5; i++) {
```

Comment:
why 5?

Reply:
just to add 5 messages and make sure the code loops over every messages